lektor_rst.py

```
@@ -15,8 +16,19 @@ def rst_to_html(text, extra_params, record):
     if ctx is None:
         raise RuntimeError('Context is required for markdown rendering')
 
+    try:
+        plugin = get_plugin('rst')
+        config = plugin.get_config()
+        settings = config.section_as_dict('docutils')
+        writer_name = settings.pop('writer', 'html')
+        extra_params.update(settings)
+    except:
```

There should be proper error handling here, not a bare except.